packages/sw-build/src/lib/generate-file-manifest.js

```diff
@@ -28,6 +30,10 @@ const errors = require('./errors');
  * generating the build manifest.
  * @param {String|Array<String>} input.globIgnores Patterns to exclude when
  * generating the build manifest.
+ * @param {String} [input.format] Either [`'iife'`](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)
```

Comment:
This is a little wordy for reference doc, could we reword to something along the lines of:

Default format is an [`'iife'`](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression), but also accepts `'es'`, which outputs an ES2015 module. 

Then open an issue to document how to use sw-build with a build process and the options available and link from this comment to those docs (once written)?